griptape/artifacts/csv_row_artifact.py

```diff
@@ -1,34 +1,28 @@
 from __future__ import annotations
 
-import csv
-import io
+from typing import Any
 
 from attrs import define, field
 
 from griptape.artifacts import BaseArtifact, TextArtifact
 
 
-@define
-class CsvRowArtifact(TextArtifact):
-    value: dict[str, str] = field(converter=BaseArtifact.value_to_dict, metadata={"serializable": True})
-    delimiter: str = field(default=",", kw_only=True, metadata={"serializable": True})
+def value_to_str(value: Any) -> str:
+    if isinstance(value, dict):
+        return "\n".join(f"{key}: {val}" for key, val in value.items())
+    else:
+        return str(value)
 
```

Comment:
do we still want to do special `dict` handling here?

Reply:
I think so. Because now the logic is isolated to this very specific Artifact rather than the much more broad `TextArtifact`.